### PR TITLE
chore(chromium): resize browser frame when emulating viewport

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/chromium/crPage.ts [hunk 1]
```diff
@@ -67,7 +68,7 @@ export class CRPage implements PageDelegate {
     this._mainFrameSession = new FrameSession(this, client, targetId);
     this._sessions.set(targetId, this._mainFrameSession);
     client.once(CRSessionEvents.Disconnected, () => this._page._didDisconnect());
-    this._pagePromise = this._mainFrameSession._initialize().then(() => this._initializedPage = this._page).catch(e => e);
+    this._pagePromise = this._mainFrameSession._initialize(!!isHeadful).then(() => this._initializedPage = this._page).catch(e => e);
```
Comment (yury-s) [inline src/chromium/crPage.ts:71]: nit: isHeadful is always defined, make it non-optional

File: src/chromium/crPage.ts [hunk 2]
```diff
@@ -364,10 +366,14 @@ class FrameSession {
     ];
   }
 
-  async _initialize() {
+  async _initialize(isHeadful: boolean) {
```
Comment (yury-s) [inline src/chromium/crPage.ts:369]: nit: can you give the parameter a name that would make more sense for headful oopif or pass actual isHeadful value in addFrameSession? It took me some time to figure out that it's not a bug in addFrameSession.

**Review by yury-s — APPROVED** [review]

File: src/chromium/crPage.ts (hunk 1 above)
Comment (pavelfeldman) [reply src/chromium/crPage.ts:71]: Done

File: src/chromium/crPage.ts (hunk 2 above)
Comment (pavelfeldman) [reply src/chromium/crPage.ts:369]: Done

**Pull request merged**